### PR TITLE
Initialize sequelize and add migrations, seeds, resolvers/schema for users/events

### PR DESCRIPTION
**Pull request by danielchen115**
## Overview

[Notion task](https://www.notion.so/uwblueprintexecs/DB-setup-bdf94d548b8d471c9b83ef29fa4374d4)

GraphQL queries for events and users that hit the database for `events`, `event(id: id)`, `users`, `user(id: id)`. Mutations are not supported yet.

## Changes
Initialize ORM and graphql connectors. Add models, migrations, seeders, schemas, resolvers for events and users.

## Screenshots
Example usage:
image

image

## Testing
Manual testing as per the screenshots. Play with it and try to break it (won't be hard for now :P, not a lot of functionality yet)!

## Other Notes 
**Setup:** 
1. Kill and restart containers to install new node packages.
2. From the node container shell (command in wiki), run:
        `sequelize db:migrate`
        `sequelize db:seed`

### CODE REVIEW

File: schema/user.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const userSchema = `
```
Comment (petorsong) [inline schema/user.js]: should events also have a resolver + schema?

File: server.js [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
+const { userResolvers } = require('./resolvers/user');
+
 const express = require('express');
 const bodyParser = require('body-parser');
 const { graphqlExpress, graphiqlExpress } = require('apollo-server-express');
 const { makeExecutableSchema } = require('graphql-tools');
 
+const { Sequelize } = require('sequelize');
+
+// TODO: this stuff can be set to env variables
```
Comment (petorsong) [inline server.js]: we should set this up in this PR, it's hardcoded in too many places right now

File: seeders/20200128023056-demo-user.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('users', [
```
Comment (petorsong) [inline seeders/20200128023056-demo-user.js]: what's the difference between a seeder and inserting an entry via migration?

File: schema/user.js (hunk 1 above)
Comment (danielchen115) [reply schema/user.js]: yep! just did the user one first to see how it works. I can do the events one tonight

File: seeders/20200128023056-demo-user.js (hunk 3 above)
Comment (danielchen115) [reply seeders/20200128023056-demo-user.js]: We may have integrity issues if we have migrations that modify tables where previous migrations already seeded data. If we separate them, we can re-seed freely without having to redo single migrations (which can be dangerous if there are more recent dependent migrations)

Also on prod/test envs, we might need to migrate without seeding

**Pull request merged**